### ISSUE TITLE
Short artifact names for bonus in campaigns

### ISSUE THREAD

**Issue by a1exsh** [ihhub/fheroes2#4421]
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.
- [X] I've checked that this issue is reproducible on the [**latest snapshot build**](https://ihhub.github.io/fheroes2/INSTALL.html#snapshots-latest-builds).

### Platform

Linux

### Describe the bug

Another one similar to #3801 

"Ballista of Quickn":
![image](https://user-images.githubusercontent.com/489601/137344819-24854b8d-d153-4a89-a488-f6f259879af8.png)

Don't know how it is called in the OG.

### Save file

[POL_0001.zip](https://github.com/ihhub/fheroes2/files/7347298/POL_0001.zip)

### Additional info

I will post more instances of the problem to this issue if I discover them.

**Comment by a1exsh** [comment]
> Let's please collect all names at once before proceeding with a fix.

We can programmatically collect names of all artifacts that are offered as a bonus in campaigns and are too long to render in this select list.  Figuring out how they were called in OG requires winning the campaign scenarios probably, or I just don't see a simpler way.  At the same time, if we don't have to use exactly the same short name as in OG, we don't have to do that.

**Comment by pflugs30** [comment]
@a1exsh Have you found any others, now that we know how to skip the hard part? :-D

**Comment by Branikolog** [comment]
Here's one more:
![image](https://user-images.githubusercontent.com/55348946/137579373-36d31497-134b-437d-8bd6-d7f355c652cc.png)

OG:
![image](https://user-images.githubusercontent.com/55348946/137579387-a55906de-c9f3-454c-9fda-5421a3d25a31.png)

**Comment by LeHerosInconnu** [comment]
Hello everyone,

I haven't tested the campaigns yet, but looking at the screenshots, you can see that for the Awards the terms should be normalized in their presentation with the first letter of some words capitalized:

"Ogre Alliance"

Instead of:

"Ogre alliance"

"Dwarf Bane"

Instead of:

"Dwarf bane"

Let me know if you'd prefer that I open another issue for this or if it's not necessary.

In fheroes2 :

> ![image](https://user-images.githubusercontent.com/55348946/137579373-36d31497-134b-437d-8bd6-d7f355c652cc.png)

**Comment by pflugs30** [comment]
I don't think we need a new issue.  That code location is right by the artifact names.  Please feel free to write any changes here, and I'll do them. Thanks!

**Comment by pflugs30** [comment]
Okay, I made some time to review the campaign awards closely (see `campaign_data.cpp`).  I also went through the OG Archibald scenario by watching a few YouTube videos (I don't have the OG on the laptop I have with me).  Here's a still-frame from [the final map](https://www.youtube.com/watch?v=ipLDDVnAbhI):
![Aplocalypse](https://user-images.githubusercontent.com/4409272/137629730-b585152f-fd81-42b3-9faa-7a30d38ff083.PNG)

I noticed the OG uses the word "DwarfBane" where we currently had "Dwarf Bane" (with a space).  I think both are inconsistent with traditional usage of the suffix "bane" (meaning, "something, typically poison, that causes death." - Oxford dictionary).  I think it's more appropriate to use a lowercase suffix with no space.  That's also how we used it in the in-game messages from the dwarves and ogres.

I also read through the rest of the code, trying to find other inconsistencies.  Here are the changes I made in the most recent changeset:

* Removed some doubles paces in text displayed to the User, favoring single spaces, in order to be consistent with most other text.
* Changed "Dwarf Bane" to "Dwarfbane" in the campaign award text to make it consistent with the in-game messages as well as to match a general pattern when using the "bane" suffix.
* Made "Alliance" lowercase to be consistent with original game and most usages.
* Changed "guild" to "Guild" since it is a proper noun and for consistency.
* Changed "Carry-over forces" to "Carry over forces" to be consistent with original game.
